mods/page_save.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+
+/**
+ * page_save module
+ *
+ * Sends the wikitext of a page to the MW API for saving
```

Comment:
I was wondering the same thing, but finally opted for another module since it really combines functionalities from the parsoid and action modules. Also, it'll keep the file sizes at bay, which makes it easier to manage. We can always send internal requests around. Thoughts?